atom/browser/api/atom_api_window.cc

```diff
@@ -786,6 +786,18 @@ bool Window::IsVisibleOnAllWorkspaces() {
   return window_->IsVisibleOnAllWorkspaces();
 }
 
+void Window::SetVibrancy(v8::Local<v8::Value> value, mate::Arguments* args) {
+  std::string type;
```

Comment:
Okay, but that means that it would remove the vibrancy to any type, other than string. Is that an acceptable behavior?